tools/clang/test/CodeGenSPIRV/pragma.pack_matrix.hlsl

```diff
@@ -0,0 +1,59 @@
+// Run: %dxc -T ps_6_0 -E main
+
+
+// CHECK: OpMemberDecorate %type_StructuredBuffer_mat2v3float 0 ColMajor
```

Comment:
SPIR-V decorate is `ColMajor` while it is `row_major` in HLSL. Is it intended?

Reply:
Yes, that's intentional. The HLSL row_major translates into ColMajor in SPIR-V, and the HLSL column_major translates into RowMajor in SPIR-V. From the [doc](https://github.com/Microsoft/DirectXShaderCompiler/blob/master/docs/SPIR-V.rst):

>Storage layout: row_major/column_major will be translated into SPIR-V ColMajor/RowMajor decoration. This is because HLSL matrix row/column becomes SPIR-V matrix column/row. If elements in a row/column are packed together, they should be loaded into a column/row correspondingly.